src/worker.rs

```diff
@@ -0,0 +1,104 @@
+use std::io::{Error, ErrorKind, Result};
+use std::sync::mpsc::Sender;
+
+use sctk::reexports::calloop::channel::Channel;
+use sctk::reexports::calloop::{channel, EventLoop};
+use sctk::reexports::calloop_wayland_source::WaylandSource;
+use sctk::reexports::client::globals::registry_queue_init;
+use sctk::reexports::client::Connection;
+
+use crate::state::{SelectionTarget, State};
+
+/// Spawn a clipboard worker, which dispatches it's own `EventQueue` each 50ms
```

Comment:
Does it dispatch every 50ms? Dispatch is called with `None`

Reply:
It doesn't.

Reply:
Ok, I just ask because the comment says that it does each 50ms. I guess maybe it was copied from before the update.

Reply:
Fixed. Also  pushed a fix for empty selection which I missed.